internal/sessiontest/helper_servers_test.go

```diff
@@ -103,19 +108,19 @@ func StartIrmaServer(t *testing.T, conf *server.Configuration) *IrmaServer {
 		conf = IrmaServerConfiguration()
 	}
 
+	mux := http.NewServeMux()
+	httpServer := httptest.NewServer(mux)
+
+	// Make sure domain is used instead of IP address.
+	conf.URL = strings.Replace(httpServer.URL, "127.0.0.1", "localhost", 1)
```

Comment:
Why is this necessary? If it is because IPv6 might be used, it would be helpful to add that to the comment above.

Reply:
Hmm, I didn't think of IPv6. The httptest library gives us an IP address instead of a domain. This causes the requestor name test to fail, because this one relies on the domain being `localhost`. I'm not sure whether we can rewrite that test in such a way that it expects an IP address. If that's not possible, we can use `url.Parse` to actually replace the host part with `localhost`. Then it should work in all cases.

Reply:
Switching the domain in the `test-requestors` scheme to a IP address does not seem to be a very smart solution, because then we hardcode the use of IPv4 even more. I'd say let's do it this way then:
`
conf.URL = fmt.Sprintf("http://localhost:%d", portFromTestServer(httpServer))
`

With portFromTestServer being:
`
func portFromTestServer(s *httptest.Server) int {
	return s.Listener.Addr().(*net.TCPAddr).Port
}
`
This function can also be used in the function `findFreePort` to prevent code duplication. The `findFreePort` function is a bit of a hack for the `RequestorServer` now. Ideally, that function shouldn't be needed.